CHANGES.txt

```diff
@@ -1,20 +1,64 @@
 Traits UI Changelog
 ===================
 
-Release 6.2.0
+Release 7.0.0
 -------------
 
+Traits 7.0.0 is a major release which focuses on modernization of the
```

Comment:
Oops, how did I overlook this in the first pass. 🤦 
This should be "TraitsUI 7.0.0", not "Traits 7.0.0"

Reply:
Arg.  This is annoying, but fixable.

Reply:
Right, I have fixed this in the documentation PR and the Github release.  The only place where it will remain is in the sdist uploaded to PyPI.  I think we can live with that.